readme.md

```diff
@@ -115,81 +115,184 @@ Case classes are serialized using the `apply` and `unapply` methods on their com
 Anything else is not supported by default, but you can add support using Custom Picklers.
 
 ## Defaults
-If a field is missing upon deserialization, weePickle uses the default value if one exists.
+If a field is missing upon **deserialization**, weePickle uses the default value if one exists:
```

Comment:
I know `deserialization` and `serialization` were already here, but they're a bit misleading. All of these behaviors have to do with case classes rather than serialization formats. For example, you can do `FromJson("{}").transform(ToMsgPack.string)`, which is both deserialization and serialization, but none of this will apply.

More README surgery would be welcome if interested. Also, totally fine to leave for a separate PR.